lib/internal/socket_list.js

```diff
@@ -22,7 +24,7 @@ SocketListSend.prototype._request = function(msg, cmd, callback) {
 
   function onclose() {
     self.slave.removeListener('internalMessage', onreply);
-    callback(new Error('Slave closed before reply'));
+    callback(new errors.Error('ERR_SLAVE_CLOSED_BEFORE_REPLY'));
```

Comment:
Are there other PR documenting this error code? Otherwise this needs a documentation.